idprovider/local/idprovider.go

```diff
@@ -28,20 +28,20 @@ import (
 	"github.com/hyperledger-labs/perun-node"
 )
 
-// Provider represents a contacts provider that provides access to contacts stored in a yaml file.
+// Provider represents an idprovider that provides access to contacts stored in a yaml file.
 //
 // It generates a cache of all contacts in the yaml file during initialization. Read, Write and Delete
 // operations act only on the cached list of contacts and do not update the yaml file.
 // The changes in cache can be updated to the yaml file by explicitly calling UpdateStorage method.
 //
 // It also stores an instance of wallet backend that will be used or decoding address strings.
 type Provider struct {
-	*contactsCache
+	*idProviderCache
 
-	contactsFilePath string
+	idProviderFilePath string
 }
 
-// New returns an instance of contacts provider to access the contacts in the given yaml file.
+// New returns an instance of idprovider to access the contacts in the given yaml file.
```

Comment:
Should we rename the function to `NewPeerIDProvider` so that it reads as `local.NewPeerIDProvider` when used in external packages ?